lighthouse-cli/sentry-prompt.ts

```diff
@@ -0,0 +1,63 @@
+/**
+ * @license Copyright 2017 Google Inc. All Rights Reserved.
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with the License. You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the specific language governing permissions and limitations under the License.
+ */
+import {Configstore, inquirer} from './shim-modules';
+
+const log = require('lighthouse-logger');
+
+const MAXIMUM_WAIT_TIME = 20 * 1000;
+// clang-format off
+const MESSAGE =
+  `${log.reset}Lighthouse is requesting permission to anonymously report back runtime exceptions.\n  ` +
```

Comment:
I'm reading this corp doc: https://goto.google.com/gpxxy on how to handle consent prompts..

Basically we need to clearly describe:

1. What this is about
2. What are the benefits to the user
3. What are the consequences to the user
4. What apps are affected
5. What is the name of this feature/thing and how can a user change their setting in the future
6. Where can the user learn more

There's some other UX suggestions which are pretty good, too. it's a nice read.


-------------

Let's create a `docs/error-reporting.md` which has some end-user-facing documentation on this. including:

  * how to change the setting (delete `~/.config/configstore/lighthouse.json` ?)  
  * the stuff you already put into the readme.
  * reported data includes test URL, its subresources, your OS, Chrome version, and Lighthouse version

And I'm thinking we tweak our consent opt-in message. (I got some inspiration from `yo`'s one)

Currently:

> Lighthouse is requesting permission to anonymously report back runtime exceptions.
> This can include data such as the test URL, its subresources, your OS, Chrome version, and Lighthouse version.
> May Lighthouse report this data to aid in improving the tool?

Proposed:

> We're constantly trying to improve Lighthouse and its reliability
> May we anonymously report runtime exceptions to improve the tool over time?
> Details: https://github.com/GoogleChrome/lighthouse/blob/master/docs/error-reporting.md

Reply:
done and done